docs/manual/fread_examples.rst

```diff
@@ -0,0 +1,490 @@
+
+    fread(data,  header=False)
+
+
+        C0	C1	C2	C3
+    0	a	b	c	d
+    1	1	2	3	4
+    2	5	6	7	8
+    3	9	10	11	12
+
+You can pass a new list of column names as well::
+
+    fread(data,  header=False, columns=["A","B","C","D"])
+
+    	A	B	C	D
+    0	a	b	c	d
+    1	1	2	3	4
+    2	5	6	7	8
+    3	9	10	11	12
+
+Row Selection
+-------------
+
+`Fread` has a `skip_to_line` parameter, where you can specify what line to read data from::
+
+    data = ('skip this line\n'
+            'a,b,c,d\n'
+            '1,2,3,4\n'
+            '5,6,7,8\n'
+            '9,10,11,12')
+
+    fread(data, skip_to_line=2)
+
+        a	b	c	d
+    0	1	2	3	4
+    1	5	6	7	8
+    2	9	10	11	12
+
+You can also skip to a line containing a particular string, and start reading data from that line. Note that `skip_to_string` and `skip_to_line` cannot be combined; you can only use one::
+
+    data = ('skip this line\n'
+            'a,b,c,d\n'
+            'first, second, third, last\n'
+            '1,2,3,4\n'
+            '5,6,7,8\n'
+            '9,10,11,12')
+
+    fread(data, skip_to_string='first')
+
+
+        first	second	third	last
+    0	1	2	3	4
+    1	5	6	7	8
+    2	9	10	11	12
+
+
+You can set the maximum number of rows to read::
+
+    data = ('a,b,c,d\n'
+            '1,2,3,4\n'
+            '5,6,7,8\n'
+            '9,10,11,12')
+
+    fread(data, max_nrows=2)
+
+
+        a	b	c	d
+    0	1	2	3	4
+    1	5	6	7	8
+
+    data = ('skip this line\n'
+            'a,b,c,d\n'
+            '1,2,3,4\n'
+            '5,6,7,8\n'
+            '9,10,11,12')
+
+    fread(data, skip_to_line=2, max_nrows=2)
+
+        a	b	c	d
+    0	1	2	3	4
+    1	5	6	7	8
+
+Setting Column Type
+--------------------
+
+You can determine the data types via the `columns` parameter::
+
+    data = ('a,b,c,d\n'
+            '1,2,3,4\n'
+            '5,6,7,8\n'
+            '9,10,11,12')
+
+    #this is useful when you are interested in only a subset of the columns
+    fread(data, columns={"a":dt.float32, "b":dt.str32})
+
+You can also pass in the data types by position::
+
+    fread(data, columns = (stype.int32, stype.str32, stype.float32))
+
+You can also change `all` the column data types with a single assignment::
+
+    fread(data, columns = dt.float32)
+
+You can change the data type for a `slice` of the columns::
+
+    #this changes the data type to float for the first three columns
+    fread(data, columns={float:slice(3)})
+
+Note that there are a small number of stypes within `datatable` (`int8`, `int16`, `int32`, `int64`, `float32`, `float64`, `str32`, `str64`, `obj64`, `bool8`)
+
+Selecting Columns
+-----------------
+
+There are various ways to select columns in `fread` :
+
+- Select with a `dictionary`::
+
+        data = ('a,b,c,d\n'
+                '1,2,3,4\n'
+                '5,6,7,8\n'
+                '9,10,11,12')
+
+        #pass Ellipsis:None to discard any columns that are not needed
+        fread(data, columns={"a":"a", Ellipsis:None})
```

Comment:
`Ellipsis` is the same as `...`, but the latter looks better: `{"a": "a", ...: None}`